src/app/helptext/vm/vm-wizard/vm-wizard.ts

```diff
@@ -77,7 +77,8 @@ volsize_validation: [Validators.required, Validators.min(1)],
 
 pool_detach_warning_paraText: T("Select zvol"),
 
-datastore_tooltip: T('Choose a pool or dataset for the new zvol.'),
+datastore_tooltip: T('Select where to create the new zvol.'),
```

Comment:
After further research, this does require a dataset, not just a directory, and does not work in a subdirectory.  So let's be specific:
```suggestion
datastore_tooltip: T('Select a dataset for the new zvol.'),
```